versions.props

```diff
@@ -95,11 +95,11 @@ ch.qos.logback:* = 1.2.11
 # Database drivers can have unexpected breaking changes and should be upgraded with caution
 com.datastax.cassandra:cassandra-driver-core = 3.8.0
 com.oracle.database.jdbc:ojdbc11 = 21.5.0.0
-com.palantir.cassandra:* = 2.2.18-1.101.0
+com.palantir.cassandra:* = 2.2.18-1.108.0
```

Comment:
Yes: this version has the new libthrift too.

Reply:
Cassandra-thrift is required for the bump, but sls-cassandra itself does not need libthrift 0.17.0 for this to be compatible, and visa versa, sls-cassandra on libthrift 0.17.0 does not require clients to be on this version

Reply:
Ah okay, it's the interface with the `cassandra-thrift` library as opposed to a wireformat thing - got it